### PR TITLE
38767

### PR DESCRIPTION
**Pull request by saikumar9**
|Field|Value|
|-----|-----|
| Redmine Ticket Number | [38767](https://devops.dchbx.org/redmine/issues/38767) |
| App-Dev Road Map # | n/a |
| Start Date | n/a |
| Due Date | n/a  |
| App-Dev Priority | UP-1 |
| Development Story Points | n/a |
| Peer Review Story Points | n/a |
| Ticket Author | Sarah Bagge |

### CODE REVIEW

File: app/views/notices/ivl/ivl_tax_cover_letter_notice.html.erb [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+    </tr>
+  </tbody>
+</table>
+
+<%= render partial: 'notices/shared/paragraph', :locals => {
+content: "This letter includes your tax Form 1095-A. You’re receiving this tax form because you or someone in your household enrolled in a private health insurance plan through #{Settings.site.short_name} in #{previous_year}."} %>
+
+<%= render partial: 'notices/shared/paragraph', :locals => {
+content: "Federal law requires most Americans to have a minimum level of health coverage. This requirement is known as the individual mandate, and it’s still in effect for the #{previous_year} and #{previous_year + 1} tax years. Your enrollment in a private health insurance plan meets this requirement. We report your enrollment status each month to the Internal Revenue Service (IRS).  These reports allow the IRS to verify the months that you were enrolled in coverage through #{Settings.site.short_name}."} %>
+
```
Comment (saipraveen18) [inline app/views/notices/ivl/ivl_tax_cover_letter_notice.html.erb]: Template changes looks good

**Comment by Kevin-Wei** [comment]
no longer needed

**Comment by Kevin-Wei** [comment]
addressed via https://github.com/dchbx/gluedb/pull/202

**Comment by sudhirreddysama** [comment]
Re-opened to re-use code for ticket #38767.

File: spec/views/notices/ivl/ivl_tax_cover_letter_notice.html.erb_spec.rb [hunk 2]
```diff
@@ -38,6 +38,9 @@
     expect(rendered).to match /This letter includes your tax Form 1095-A. You’re receiving this tax form because you or someone/
     expect(rendered).to match /If you have questions or concerns, we’re here to help./
     expect(rendered).to match /The #{Settings.site.short_name} Team/
+    expect(rendered).to match /#{Settings.site.ivl_responsibility_url}/
```
Comment (saikumar9) [inline spec/views/notices/ivl/ivl_tax_cover_letter_notice.html.erb_spec.rb]: Good Test coverage.

File: app/views/notices/ivl/ivl_tax_cover_letter_notice.html.erb [hunk 3]
```diff
@@ -137,7 +137,7 @@
...
 
 <%= render partial: 'notices/shared/paragraph', :locals => {
-content: "Federal law requires most Americans to have a minimum level of health coverage. This requirement is known as the individual mandate, and it’s still in effect for the #{previous_year} and #{previous_year + 1} tax years. Your enrollment in a private health insurance plan meets this requirement. We report your enrollment status each month to the Internal Revenue Service (IRS).  These reports allow the IRS to verify the months that you were enrolled in coverage through #{Settings.site.short_name}."} %>
+content: "Federal law required most Americans to have a minimum level of health coverage or pay a tax penalty through #{previous_year}. The #{Settings.aca.state_name} now has an individual responsibility requirement. This means that DC residents must have qualifying health coverage all year. If you don't have coverage, and don't qualify for an exemption, you’ll have to pay a penalty when you file your DC taxes. For more information: <a href= #{Settings.site.ivl_responsibility_url}>https://dchealthlink.com/individual-responsibility-requirement. </a>"} %>
```
Comment (saikumar9) [inline app/views/notices/ivl/ivl_tax_cover_letter_notice.html.erb]: Made changes as per updated **2018** template.

**Pull request merged**